modules/gitops-modifying-keycloak-requests-limits.adoc

```diff
@@ -27,5 +27,5 @@ $ oc -n openshift-gitops patch argocd openshift-gitops --type='json' -p='[{"op":
 
 [NOTE]
 ====
-Keycloak created by the {gitops-title} only persists the changes that are made by the operator. If the Keycloak restarts, any additional configuration created by the Admin in Keycloak is deleted.
+Keycloak created by the {gitops-title} only persists the changes that are made by the operator. If the Keycloak restarts, any additional configuration created by the administrator in Keycloak is deleted.
```

Comment:
Please also include change the following phrase to ensure better readability:

"the {gitops-title}" -> "the {gitops-title} Operator"

Reply:
Also change "made by the operator" to "made by the Operator"